<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowRecipeWindow</class>
 <widget class="QMainWindow" name="ShowRecipeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recipe List</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Resources/meadappico_Uh1_icon.ico</normaloff>../Resources/meadappico_Uh1_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="ConfirmBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Open</set>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>551</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
